SQL/New_patches/2018-10-01-Add_data_release_permissions.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO permissions (code,description,categoryID) VALUES ('data_release_upload','Data Release: Upload file',2), ('data_release_edit_permissions','Data Release: Give user permission to view files',2);
```

Comment:
Same here. Edit and view, not just view.

Reply:
see comment above